extensions/flight-sql/build.gradle

```diff
@@ -0,0 +1,78 @@
+plugins {
+    id 'java-library'
+    id 'io.deephaven.project.register'
+}
+
+description = 'The Deephaven Flight SQL library'
+
+sourceSets {
+    jdbcTest {
+        compileClasspath += sourceSets.main.output
+        runtimeClasspath += sourceSets.main.output
+    }
```

Comment:
*Need*, no. I do think there is structural value in having them separate though. I have this comment

```gradle
    // JDBC testing needs an actually server instance bound to a port because it can only connect over JDBC URIs like
    // jdbc:arrow-flight-sql://localhost:1000.
    jdbcTestImplementation project(':server-jetty')
```

which is my primary motivation.